modules/microshift-auto-apply-manifests.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-auto-apply-manifests_{context}"]
+= Automatically applying manifests with `kustomize`
+
+When you run the `kustomize` feature, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k` command to apply the manifests to the cluster. Providing multiple directories allows a flexible method of managing {product-title} workloads.
```

Comment:
```suggestion
When you run `kustomize`, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k` command to apply the manifests to the cluster. Providing multiple directories allows a flexible method of managing {product-title} workloads.
```

kustomize is a separate command line tool. It's not a feature of MicroShift, which is implied by this new wording.

Reply:
This is something for the next PR.

Reply:
Providing multiple directories allows a flexible method of managing {product-title} workloads.--> this is a small nit, but I was thinking this sentence maybe should be the intro sentence to this para? Also, IBMSG Global Audience recommendation p. 55 to watch verb phrases at the beginning of a sentence and p. 77 to watch out for Anthropomorphism, I do think this could be clearer and more user focused, perhaps something like this: "When you run the `kustomize` feature, automatic searching of the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file occurs. Once the file is located, the `kubectl apply -k` command automatically runs, and the manifest is applied to the cluster" This suggestion has some passive voice meh. Anyways take what you will and make it your own.